wg-opcodes/summaries/current-status.md

```diff
@@ -0,0 +1,56 @@
+# Metadata
+
+Title:            Current Status of OpCodes Workgroup
+Workgroup:        OpCodes
+Document Version: 1.0
+
+# Document Purpose
+
+This document is intended to provide a single source of truth of the status of ongoing op codes
+discussions relative to the upcoming hardfork. Any discussions not related to the latest upcoming
+fork will be referenced and postponed until the next hardfork or removed entirely if the discussion
+is in a complete form and no spec is planned to be proposed.  Implementation details or specs of
+particular op codes is out of scope of this document.
+
+# Current Status of OpCodes
+
+## OpCodes agreed upon for the next upcoming hard fork (May 2018)
+
+May 2018 hard fork spec proposal (for reference): https://github.com/bitcoincashorg/spec/pull/53/files
+
+OP_AND
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_BIN2NUM
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_CAT
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_DIV
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_MOD
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_NUM2BIN
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_OR
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_SPLIT
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+OP_XOR
+ * Spec proposed: https://github.com/bitcoincashorg/spec/pull/54/files
+
+## OpCodes Under Discussion or Previously Discussed
```

Comment:
Need to add OP_MUL and OP_INVERT.  Still need to dig up discussions on these, but it looks like OP_MUL is tentative for the Nov 2018 fork and OP_INVERT may be unnecessary.

Reply:
Not sure why op_invert is unnecessary?  It was put off from May to limit scope to a manageable size but not heard anyone say it's unnecessary.

Reply:
Someone had suggested that OP_INVERT may not be necessary with the inclusion of OP_REPEAT or OP_BIN2NUM.  I haven't really verified these claims though.  I'll go ahead and include OP_INVERT for later investigation and not mention the unnecessary bit.